Bindings/Python/tests/test_opensense.py

```diff
@@ -33,6 +33,10 @@ def test_createObjects(self):
         constraint_var = .0001
         ikSolver = osim.InverseKinematicsSolver(model, mRefs, oRefs, coordinateReferences, constraint_var)
         print("Created InverseKinematicsSolver object..")
+        oRefs = osim.BufferedOrientationsReference()
+        print("Created BufferedOrientationsReference object..")
+        ikSolver = osim.InverseKinematicsSolver(model, mRefs, oRefs, coordinateReferences, constraint_var)
+        print("Created InverseKinematicsSolver object with BufferedOrientationsReference..")
 
```

Comment:
Good exercise of the constructor, thank you. Can we also add a line to test adding/appending data to the oRefs in order to exercise appending data to be solved downstream? Thank you

Reply:
Absolutely. I'll add that test in a bit.

Reply:
Okay, I have added the test.